### PR TITLE
Add tini bin from openshift origin-release image

### PR DESCRIPTION
**Pull request by sabinaaledort**
Adding the tini bin from the openshift origin-release image to FRR container Dockerfile.

### CODE REVIEW

File: Dockerfile.openshift [hunk 1]
```diff
@@ -7,6 +7,8 @@
 FROM registry.ci.openshift.org/ocp/builder:rhel-8-base-openshift-4.10
 USER root
 
+FROM openshift/origin-release:golang-1.16 as builder-tini
```
Comment (fedepaol) [inline Dockerfile.openshift:10]: I'd put this above the previous FROM

**Comment by msherif1234** [comment]
lets hold this ART team they will add tini to rhel-8-base-openshift-4.10 , also if we switch to multi stage build there will be additional changes needed to ART side https://github.com/openshift/ocp-build-data/blob/openshift-4.10/images/ose-frr.yml#L16-L17

**Comment by sabinaaledort** [comment]
> lets hold this ART team they will add tini to rhel-8-base-openshift-4.10 , also if we switch to multi stage build there will be additional changes needed to ART side https://github.com/openshift/ocp-build-data/blob/openshift-4.10/images/ose-frr.yml#L16-L17

ok

**Comment by sabinaaledort** [comment]
was fixed in another commit

**Pull request closed without merging**